packages/clerk-js/src/ui/components/Subscriptions/SubscriptionsList.tsx

```diff
@@ -15,7 +31,19 @@ export function SubscriptionsList() {
   };
 
   return (
-    <Table tableHeadVisuallyHidden>
+    <Table
+      tableHeadVisuallyHidden
+      sx={t => ({
+        'tr > td': {
+          paddingInline: t.sizes.$4,
+          paddingBlock: t.sizes.$2,
+        },
+        'tr > th': {
+          paddingInline: t.sizes.$4,
+          paddingBlock: t.sizes.$2,
+        },
```

Comment:
Are these not the defaults already? https://github.com/clerk/javascript/blob/0ae040303d239b75a3221436354a2c2ecdb85aae/packages/clerk-js/src/ui/primitives/Table.tsx#L38-L41